### ISSUE TITLE
Investigate how to properly clean up tooltips

### ISSUE THREAD

**Issue by zachmullen** [girder/girder#695]
Maybe wrap `$.fn.tooltip`, or clean up on route change, something like that.

ping @manthey

**Comment by jbeezley** [comment]
Out of curiosity, is this a general bootstrap tooltip requirement or is it only related to the girder framework?  I was under the impression that you just needed to delete the DOM element it is attached to.

**Comment by manthey** [comment]
Currently, when we change views via the Backbone router, we remove the .tooltip elements.  This isn't always desirable, as in the XData project, I was using a bootstrap plugin that expected its .tooltip to persist.  I speculate that we aren't assigning the tooltips to the proper container (or something) when they are created, which is why they weren't being cleaned up.

The problem occurred when you had a tooltip showing and asked the browser to go back, and our work around of removing the .tooltip element seemed sufficient until this week.

**Comment by zachmullen** [comment]
Yeah, we should check out how we are using the `container` property of the tooltip.

**Comment by jbeezley** [comment]
I see.  I'll be interested in hearing what you find out.

**Comment by danlamanna** [comment]
@brianhelba Did any of the recent work with tooltips resolve this?
